lib/pure/unittest.nim

```diff
@@ -570,6 +573,7 @@ template test*(name, body) {.dirty.} =
       )
       testEnded(testResult)
       checkpoints = @[]
+    {.warning[BareExcept]:on.}
```

Comment:
if someone had specified `BareExcept:off` on the command line, won't this override the command line setting?

Reply:
push/pop, when it works - it sometimes doesn't or infects other modules, not sure what the bugs are